internal/view/content.html

```diff
@@ -63,18 +63,34 @@
 						ShowId = (typeof opts.config.ShowId === "boolean") ? opts.config.ShowId : false,
 						ListMode = (typeof opts.config.ListMode === "boolean") ? opts.config.ListMode : false,
 						NightMode = (typeof opts.config.NightMode === "boolean") ? opts.config.NightMode : false,
-						UseArchive = (typeof opts.config.UseArchive === "boolean") ? opts.config.UseArchive : false;
+                        Theme = (typeof opts.config.Theme === "string" && opts.config.Theme !== "") ? opts.config.Theme : "follow",
```

Comment:
This seems to have different whitespace type than the other lines (probably spaces instead of tabs?)

Reply:
`make style` only renders the less files into css, it does not modify the html/js files. What I have setup is auto formatting on save on my editor.